### DISCUSSION TITLE
TypeError: localErrorHandlerData.handle is not a function

### DISCUSSION THREAD

**Discussion by gardenjoon** [serverless/serverless#10075] [Q&A]
i've got this error message but i can't find any documents on Google.
so please can you know about this error?

this error caused by

> serverless deploy 

when deploied, i found makeReview was get method so i have to change it post method
deleted lambda function and apiGateway resource 
and deploied again, i got this error

error message below 
======================================================
> garden@Gardens-mac service % sls deploy fucntion -f makeReview

```
Serverless: Running "serverless" installed locally (in service node_modules)

 Serverless Warning --------------------------------------

  CLI options definitions were upgraded with "type" property (which could be one of "string", "boolean", "multiple"). Below listed plugins do not predefine type for introduced options:
   - ServerlessOffline for "apiKey", "corsAllowHeaders", "corsAllowOrigin", "corsDisallowCredentials", "corsExposedHeaders", "disableCookieValidation", "enforceSecureCookies", "hideStackTraces", "host", "httpPort", "httpsProtocol", "lambdaPort", "noPrependStageInUrl", "noAuth", "ignoreJWTSignature", "noTimeout", "prefix", "printOutput", "resourceRoutes", "useChildProcesses", "useWorkerThreads", "websocketPort", "webSocketHardTimeout", "webSocketIdleTimeout", "useDocker", "layersDir", "dockerReadOnly", "functionCleanupIdleTimeSeconds", "allowCache"

  Please report this issue in plugin issue tracker.

/opt/homebrew/Cellar/serverless/2.60.1/libexec/lib/node_modules/serverless/node_modules/essentials/index.js:12
		throw reason;
		^
typeError: localErrorHandlerData.handle is not a function
    at module.exports (/opt/homebrew/Cellar/serverless/2.60.1/libexec/lib/node_modules/serverless/lib/cli/handle-error.js:108:29)
    at process.<anonymous> (/opt/homebrew/Cellar/serverless/2.60.1/libexec/lib/node_modules/serverless/scripts/serverless.js:51:3)
    at Object.onceWrapper (node:events:510:26)
    at process.emit (node:events:402:35)
    at process.emit (node:domain:475:12)
    at process._fatalException (node:internal/process/execution:170:25)/
```

**Comment by pgrzesik** [comment]
Hello, it looks like you might not be on the latest version of the Framework - could you please upgrade to the latest version and check if the problem persists?

**Comment by jcperez** [comment]
Can you share your `serverless.yml` file?

I was having the same issue, but accidentally we changed the location (added an extra tab) of the following section within the `serverless.yml`. After fixing it, the error was gone.

```
plugins:
  - serverless-offline
```

**Comment by playdaygameschris** [comment]
I got the exact same problem too.
I am trying to npm install in arm64 node14 container, and deploy it to lambda arm64 function
Tried a lot of sls versions like 2.65,2.60.2, 2.61
none of above works

**Reply by pgrzesik** [reply]
Could you please report it with a reproducible example as issue? It was a problem previously but it has been addressed a while ago and definitely should not appear in 2.65.0

**Reply by aaronlevenstein** [reply]
I am getting the same issue in sls version 2.66.0
aaronlevenstein@AaronlesteinPro bwell_autot % sls -v
Serverless: Running "serverless" installed locally (in service node_modules)
Framework Core: 2.66.0 (local)
Plugin: 5.5.1
SDK: 4.3.0
Components: 3.17.2

**Reply by pgrzesik** [reply]
In this particular case you might need to update the globally installed version of the Framework as well - I see you're using locally installed version in above case, but the handler error happens in global version that falls back to local one
